### DISCUSSION TITLE
How to use CDN links?

### DISCUSSION THREAD

**Discussion by Bobakanoosh** [histoire-dev/histoire#284] [Q&A] [answered]
I load icons from fontawesome via a url in my `index.html`. I haven't been able to find anything in docs/previous issues, aside from someone using font awesome as a package.

**Comment by botmaster** [comment, marked as answer]
Hello
Try this in `histoire.setup.js`

```// Add Fontawesome.
const head = document.getElementsByTagName("head")[0];
const script = document.createElement("script");
script.type = "text/javascript";
script.src = "https://kit.fontawesome.com/xxxxxxxx.js";
script.crossOrigin = "anonymous";
head.appendChild(script);

**Reply by Bobakanoosh** [reply]
Thank you! Didn't realize `histoire.setup.ts` was outside of vite and thus loaded into the actual code when running `story:dev`, makes sense though
